ql.mm

```diff
@@ -10420,54 +9896,53 @@ Kalmbach axioms (soundness proofs) that require WOML
     BUSEZEZEUMUQUREZEZURVJVMUMUQBUSKMVMVNUMVLURUQVLBULURDZEBULBUKEZDZEZURUSVPBU
     RULNMVPVRBURVQULUKBIOMVSVQURBUKPBUKISQMMVOURUMURVNUKULBTUQURURUOURUPBAUKUAU
     KBBTRURUBRRURVNUMURUQUCUDUEQQQVBUNVCUTABUFABUGUHABUIUJ $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 3, and j is set to 0.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 3, and j is set to 0.  (Contributed by Roy F. Longton, 1-Jul-2005.)
+     (Revised by Roy F. Longton, 3-Jul-2005.) $)
   lem4.6.6i3j0 $p |- ( ( a ->3 b ) v ( a ->0 b ) ) = ( a ->0 b ) $=
     ( wn wa wi3 wi0 ax-a3 ax-r1 lor ax-a2 omln ax-r2 ax-r5 leid leor lel2or leo
     wo leao1 3tr lebi df-le2 df-i3 df-i0 2or 3tr1 ) ACZBDZUGBCZDZRZAUGBRZDZRZUL
     RZULABEZABFZRUQUOUKUMULRZRUKUMUGRZBRZRZULUKUMULGURUTUKUTURUMUGBGHIVAUKULBRZ
     RUKULRULUTVBUKUSULBUSUGUMRULUMUGJABKLMIVBULUKVBULULULBULNBUGOPULBQUAIUKULUH
     ULUJUGBBSUGUIBSPUBTTUPUNUQULABUCABUDZUEVCUF $.
-    $( [3-Jul-05] $) $( [1-Jul-05] $)
 
   $( Equation 4.14 of [MegPav2000] p. 23.  The variable i in the paper is set
-     to 3, and j is set to 1.  (Contributed by Roy F. Longton, 3-Jul-05.) $)
+     to 3, and j is set to 1.  (Contributed by Roy F. Longton, 1-Jul-2005.)
```

Comment:
The Contributed line should be 3-Jul-2005 and there should be no Revised by. I realize that the original has both dates but there's no point in having two dates two days apart.